### PR TITLE
Om trans2

### PR DESCRIPTION
**Pull request by om64**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Fixed tickets | comma-separated list of tickets fixed by the PR, if any

Correcting platform.fr.yml and translating it into Dutch.

### CODE REVIEW

File: main/core/Resources/translations/platform.fr.yml [hunk 1]
```diff
@@ -202,7 +202,7 @@ create_resource: Créer une nouvelle ressource
 create_role: Créer un rôle
 create_security_token: Créer un jeton d'authentification
 create_subcategory: Créer une sous-catégorie
-create_user: Créer un utilisateur
+create_user: Créer un utilisateurcd tr
```
Comment (purplefish32) [inline main/core/Resources/translations/platform.fr.yml]: This looks strange

**Comment by om64** [comment]
@purplefish32 The file has been corrected. Thanks for seeing my mistake.

**Comment by kitan1982** [comment]
LGTM

**Comment by ngodfraind** [comment]
LGTM

**Comment by purplefish32** [comment]
LGTM

**Comment by om64** [comment]
@purplefish32 How? What are the command lines?

**Comment by om64** [comment]
Thanks a lot @ngodfraind

**Pull request merged**